src/main/java/com/cleanroommc/groovyscript/compat/mods/draconicevolution/EnergyCore.java

```diff
@@ -0,0 +1,131 @@
+package com.cleanroommc.groovyscript.compat.mods.draconicevolution;
+
+import com.brandon3055.brandonscore.lib.MultiBlockStorage;
+import com.brandon3055.draconicevolution.world.EnergyCoreStructure;
+import com.cleanroommc.groovyscript.api.GroovyBlacklist;
+import com.cleanroommc.groovyscript.api.GroovyLog;
+import com.cleanroommc.groovyscript.api.IScriptReloadable;
+import com.cleanroommc.groovyscript.api.documentation.annotations.Example;
+import com.cleanroommc.groovyscript.api.documentation.annotations.MethodDescription;
+import com.cleanroommc.groovyscript.api.documentation.annotations.RegistryDescription;
+import com.cleanroommc.groovyscript.core.mixin.draconicevolution.EnergyCoreStructureAccessor;
+import com.cleanroommc.groovyscript.core.mixin.draconicevolution.MultiBlockStorageAccessor;
+import com.cleanroommc.groovyscript.helper.Alias;
+import net.minecraft.block.Block;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.fml.common.registry.ForgeRegistries;
+import org.jetbrains.annotations.ApiStatus;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+@RegistryDescription
+public class EnergyCore implements IScriptReloadable {
+
+    private static final String DRACONIUM = "draconicevolution:draconium_block";
+    private static final String DRACONIC = "draconicevolution:draconic_block";
+    private static final String REDSTONE = "minecraft:redstone_block";
+
+    private int version = 0;
+    private MultiBlockStorage[] original;
+    private MultiBlockStorage[] edited;
+    private String[] inner;
+    private String[] outer;
+
+    private void init() {
+        if (this.original != null) return;
+        EnergyCoreStructure ecs = new EnergyCoreStructure();
+        ecs.initialize(null);
+        this.original = ((EnergyCoreStructureAccessor) ecs).getStructureTiers();
+        this.edited = Arrays.copyOf(this.original, this.original.length);
+        this.inner = new String[this.original.length];
+        this.outer = new String[this.original.length];
+        onReload(); // increases version to 1
+    }
+
+    @Override
+    public Collection<String> getAliases() {
+        return Alias.generateOfClass(EnergyCore.class);
+    }
+
+    @Override
+    public void onReload() {
```

Comment:
the video uploaded on discord as a showcase clearly shows reloading functioning, but i cannot reproduce this behavior. on my computer, after a change reloading does not cause the multiblock to become invalidated, and requires a full restart to see the affect.